rust-libretro/src/environment.rs

```diff
@@ -1869,13 +1974,11 @@ pub unsafe fn set_content_info_override(
 ///   [`Core::on_load_game_special`]
 #[proc::context(LoadGameContext)]
 #[proc::context(LoadGameSpecialContext)]
-pub unsafe fn get_game_info_ext(callback: retro_environment_t) -> Option<retro_game_info_ext> {
```

Comment:
This is supposed to return an array. https://github.com/libretro/RetroArch/blob/master/libretro-common/include/libretro.h#L1477-L1510 Or I guess in this case, a slice; the frontend maintains ownership of the underlying `retro_game_info_ext` array, so it'll be fine.